Procfile

```diff
@@ -1,2 +1,2 @@
 release: python manage.py migrate --noinput
-web: bin/start-nginx gunicorn -c gunicorn.conf pydotorg.wsgi
+web: python manage.py purge_fastly_cache && bin/start-nginx gunicorn -c gunicorn.conf pydotorg.wsgi
```

Comment:
I think we may need to move this into a `run.sh` in case `Procfile` doesn't support `&&`.

Reply:
kinda stinks that normal heroku apps don't have a "post deploy" phase.

it'd be ideal to issue the command after the app is up and running. this solution is a bit prone to a race condition, as the purge is issued before the app is up. traffic may still get the old app.

Reply:
Yes, unfortunately I couldn't find a way to do that. I'm not sure if changing the order would help.